chainer/functions/math/logsumexp.py

```diff
@@ -1,6 +1,7 @@
 import chainer
 from chainer.backends import cuda
 from chainer import function_node
+from chainer.utils import force_array
```

Comment:
Could you import the module rather than importing the function?
```
from chainer import utils
...
        y = utils.force_array(x - m)
```

Reply:
Ok, I will fix!